pom.xml

```diff
@@ -21,7 +21,7 @@
     <maven.compiler.release>11</maven.compiler.release>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <quarkus.version>2.16.2.Final</quarkus.version>
+    <quarkus.version>3.0.0.Alpha4</quarkus.version>
```

Comment:
Isn’t the attributes.adoc file also changed automatically during the build?

Reply:
It didn’t change when doing a local build. If I undesrtand this correctly, it is based on the release version, which is still unchanged.

Reply:
Ah you're right. Except if the attributes.adoc looks like https://github.com/quarkiverse/quarkus-prettytime/blob/main/docs/templates/includes/attributes.adoc